### ISSUE TITLE
[Bug]: many `i/o timeout` during bvt test

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#6531]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): b9c225f4
- Hardware parameters:
- OS type:
- Others: related to #6423 and #6209
```

### Actual Behavior

```json
{"level":"INFO","time":"2022/11/12 18:12:39.529665 +0000","caller":"frontend/util.go:510","msg":"rowCount 2 \ntime of getDataFromPipeline : 10.458µs \nprocessBatchTime 7.154µs \nrow2colTime 0s \nallocateOutBufferTime 344ns \noutputQueue.flushTime 0s \nprocessBatchTime - row2colTime - allocateOutbufferTime - flushTime 6.81µs \nrestTime(=tTime - row2colTime - allocateOutBufferTime) 10.114µs \nprotoStats flushCount 0 writeCount 1 \nwriteBytes 0 0 MB\n connectionId 1011"}
{"level":"INFO","time":"2022/11/12 18:12:39.529761 +0000","caller":"frontend/util.go:510","msg":"time of Exec.Run : 818.933µs connectionId 1011"}
{"level":"INFO","time":"2022/11/12 18:12:39.529846 +0000","caller":"frontend/util.go:495","msg":"query trace status","connection_id":1011,"statement":"show tables","status":"success","session_info":"connectionId 1011"}
{"level":"INFO","time":"2022/11/12 18:12:39.530000 +0000","caller":"frontend/util.go:495","msg":"query trace","connection_id":1011,"query":"select \n    \t\t\t\t\t\ttask_id,\n\t\t\t\t\t\t\ttask_metadata_id,\n\t\t\t\t\t\t\ttask_metadata_executor,\n\t\t\t\t\t\t\ttask_metadata_context,\n\t\t\t\t\t\t\ttask_metadata_option,\n\t\t\t\t\t\t\ttask_parent_id,\n\t\t\t\t\t\t\ttask_status,\n\t\t\t\t\t\t\ttask_runner,\n\t\t\t\t\t\t\ttask_epoch,\n\t\t\t\t\t\t\tlast_heartbeat,\n\t\t\t\t\t\t\tresult_code,\n\t\t\t\t\t\t\terror_msg,\n\t\t\t\t\t\t\tcreate_at,\n\t\t\t\t\t\t\tend_at \n\t\t\t\t\t\tfrom mo_task.sys_async_task where task_id>0 AND task_runner='00000000-0000-0000-0000-200000000000' order by task_id limit 1","session_info":"connectionId 1011"}
{"level":"INFO","time":"2022/11/12 18:12:39.585850 +0000","caller":"frontend/util.go:510","msg":"time of Exec.Build : 55.741969ms connectionId 1011"}
{"level":"INFO","time":"2022/11/12 18:12:39.646889 +0000","caller":"frontend/util.go:510","msg":"time of Exec.Run : 60.619927ms connectionId 1011"}
{"level":"INFO","time":"2022/11/12 18:12:39.647160 +0000","caller":"frontend/util.go:495","msg":"query trace status","connection_id":1011,"statement":"select sys_async_task.task_id, sys_async_task.task_metadata_id, sys_async_task.task_metadata_executor, sys_async_task.task_metadata_context, sys_async_task.task_metadata_option, sys_async_task.task_parent_id, sys_async_task.task_status, sys_async_task.task_runner, sys_async_task.task_epoch, sys_async_task.last_heartbeat, sys_async_task.result_code, sys_async_task.error_msg, sys_async_task.create_at, sys_async_task.end_at from mo_task.sys_async_task where sys_async_task.task_id > 0 and sys_async_task.task_runner = 00000000-0000-0000-0000-200000000000 order by sys_async_task.task_id limit 1","status":"success","session_info":"connectionId 1011"}
{"level":"INFO","time":"2022/11/12 18:12:49.647782 +0000","caller":"frontend/util.go:495","msg":"query trace","connection_id":1011,"query":"use mo_task","session_info":"connectionId 1011"}
[mysql] 2022/11/12 18:12:54 packets.go:37: read tcp 10.0.124.39:35212->10.0.124.39:6001: i/o timeout
{"level":"ERROR","time":"2022/11/12 18:12:54.648462 +0000","name":"cn-service.task-runner","caller":"taskservice/task_runner.go:283","msg":"fetch task failed","uuid":"00000000-0000-0000-0000-200000000000","runner-id":"00000000-0000-0000-0000-200000000000","error":"invalid connection"}
{"level":"INFO","time":"2022/11/12 18:13:02.224149 +0000","caller":"frontend/util.go:510","msg":"check password succeeded connectionId 1012"}
{"level":"INFO","time":"2022/11/12 18:13:02.224330 +0000","caller":"frontend/util.go:495","msg":"query trace","connection_id":1012,"query":"use mo_task","session_info":"connectionId 1012"}
{"level":"INFO","time":"2022/11/12 18:13:04.651432 +0000","caller":"frontend/util.go:510","msg":"check password succeeded connectionId 1013"}
{"level":"INFO","time":"2022/11/12 18:13:04.651603 +0000","caller":"frontend/util.go:495","msg":"query trace","connection_id":1013,"query":"use mo_task","session_info":"connectionId 1013"}
[mysql] 2022/11/12 18:13:09 packets.go:37: read tcp 10.0.124.39:39096->10.0.124.39:6001: i/o timeout
{"level":"ERROR","time":"2022/11/12 18:13:09.651720 +0000","name":"cn-service.task-runner","caller":"taskservice/task_runner.go:283","msg":"fetch task failed","uuid":"00000000-0000-0000-0000-200000000000","runner-id":"00000000-0000-0000-0000-200000000000","error":"invalid connection"}
{"level":"INFO","time":"2022/11/12 18:13:10.132591 +0000","caller":"frontend/util.go:510","msg":"User 36a76541-93c4-41a4-9b43-a0ac0c0ca4c4 change database from [mo_task] to [mo_task] connectionId 1011"}
{"level":"ERROR","time":"2022/11/12 18:13:10.132698 +0000","caller":"frontend/util.go:505","msg":"query trace status","connection_id":1011,"statement":"use mo_task","status":"fail","error":"internal error: routine send response failed. error:illegal state ","session_info":"connectionId 1011","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logError\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:505\ngithub.com/matrixorigin/matrixone/pkg/frontend.logStatementStringStatus\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:489\ngithub.com/matrixorigin/matrixone/pkg/frontend.logStatementStatus\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:481\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3008\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3141\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:160"}
{"level":"ERROR","time":"2022/11/12 18:13:10.132904 +0000","caller":"frontend/util.go:520","msg":"routine send response failed &{1 0 3 0xc006643080 0 0 0}. error:illegal state  connectionId 1011","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logErrorf\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:520\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:166"}
{"level":"ERROR","time":"2022/11/12 18:13:10.132962 +0000","caller":"frontend/util.go:520","msg":"rollback the txn. connectionId 1011","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logErrorf\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:520\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:180"}
{"level":"INFO","time":"2022/11/12 18:13:10.135776 +0000","caller":"frontend/util.go:510","msg":"User 36a76541-93c4-41a4-9b43-a0ac0c0ca4c4 change database from [] to [mo_task] connectionId 1012"}
{"level":"ERROR","time":"2022/11/12 18:13:10.135982 +0000","caller":"frontend/util.go:505","msg":"query trace status","connection_id":1012,"statement":"use mo_task","status":"fail","error":"internal error: routine send response failed. error:illegal state ","session_info":"connectionId 1012","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logError\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:505\ngithub.com/matrixorigin/matrixone/pkg/frontend.logStatementStringStatus\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:489\ngithub.com/matrixorigin/matrixone/pkg/frontend.logStatementStatus\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:481\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3008\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3141\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:160"}
{"level":"ERROR","time":"2022/11/12 18:13:10.136076 +0000","caller":"frontend/util.go:520","msg":"routine send response failed &{1 0 3 0xc006643950 0 0 0}. error:illegal state  connectionId 1012","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logErrorf\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:520\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:166"}
{"level":"INFO","time":"2022/11/12 18:13:10.136089 +0000","caller":"frontend/util.go:510","msg":"User 36a76541-93c4-41a4-9b43-a0ac0c0ca4c4 change database from [] to [mo_task] connectionId 1013"}
{"level":"ERROR","time":"2022/11/12 18:13:10.136140 +0000","caller":"frontend/util.go:520","msg":"rollback the txn. connectionId 1012","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logErrorf\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:520\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:180"}
{"level":"ERROR","time":"2022/11/12 18:13:10.136157 +0000","caller":"frontend/util.go:505","msg":"query trace status","connection_id":1013,"statement":"use mo_task","status":"fail","error":"internal error: routine send response failed. error:illegal state ","session_info":"connectionId 1013","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logError\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:505\ngithub.com/matrixorigin/matrixone/pkg/frontend.logStatementStringStatus\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:489\ngithub.com/matrixorigin/matrixone/pkg/frontend.logStatementStatus\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:481\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3008\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3141\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:160"}
{"level":"ERROR","time":"2022/11/12 18:13:10.136212 +0000","caller":"frontend/util.go:520","msg":"routine send response failed &{1 0 3 0xc007fcd4a0 0 0 0}. error:illegal state  connectionId 1013","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logErrorf\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:520\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:166"}
{"level":"ERROR","time":"2022/11/12 18:13:10.136234 +0000","caller":"frontend/util.go:520","msg":"rollback the txn. connectionId 1013","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logErrorf\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:520\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:180"}
{"level":"INFO","time":"2022/11/12 18:13:19.654246 +0000","caller":"frontend/util.go:510","msg":"check password succeeded connectionId 1014"}
{"level":"INFO","time":"2022/11/12 18:13:19.654504 +0000","caller":"frontend/util.go:495","msg":"query trace","connection_id":1014,"query":"use mo_task","session_info":"connectionId 1014"}
{"level":"INFO","time":"2022/11/12 18:13:19.658593 +0000","caller":"frontend/util.go:510","msg":"User 36a76541-93c4-41a4-9b43-a0ac0c0ca4c4 change database from [] to [mo_task] connectionId 1014"}
{"level":"INFO","time":"2022/11/12 18:13:19.658739 +0000","caller":"frontend/util.go:495","msg":"query trace status","connection_id":1014,"statement":"use mo_task","status":"success","session_info":"connectionId 1014"}
{"level":"INFO","time":"2022/11/12 18:13:19.658824 +0000","caller":"frontend/util.go:495","msg":"query trace","connection_id":1014,"query":"show tables","session_info":"connectionId 1014"}
{"level":"INFO","time":"2022/11/12 18:13:19.662683 +0000","caller":"frontend/util.go:510","msg":"time of Exec.Build : 3.794927ms connectionId 1014"}
{"level":"INFO","time":"2022/11/12 18:13:19.663223 +0000","caller":"frontend/util.go:510","msg":"rowCount 2 \ntime of getDataFromPipeline : 6.61µs \nprocessBatchTime 3.942µs \nrow2colTime 0s \nallocateOutBufferTime 232ns \noutputQueue.flushTime 0s \nprocessBatchTime - row2colTime - allocateOutbufferTime - flushTime 3.71µs \nrestTime(=tTime - row2colTime - allocateOutBufferTime) 6.378µs \nprotoStats flushCount 0 writeCount 1 \nwriteBytes 0 0 MB\n connectionId 1014"}
{"level":"INFO","time":"2022/11/12 18:13:19.663252 +0000","caller":"frontend/util.go:510","msg":"time of Exec.Run : 489.938µs connectionId 1014"}
{"level":"INFO","time":"2022/11/12 18:13:19.663311 +0000","caller":"frontend/util.go:495","msg":"query trace status","connection_id":1014,"statement":"show tables","status":"success","session_info":"connectionId 1014"}
{"level":"INFO","time":"2022/11/12 18:13:19.663755 +0000","caller":"frontend/util.go:495","msg":"query trace","connection_id":1014,"query":"select \n    \t\t\t\t\t\ttask_id,\n\t\t\t\t\t\t\ttask_metadata_id,\n\t\t\t\t\t\t\ttask_metadata_executor,\n\t\t\t\t\t\t\ttask_metadata_context,\n\t\t\t\t\t\t\ttask_metadata_option,\n\t\t\t\t\t\t\ttask_parent_id,\n\t\t\t\t\t\t\ttask_status,\n\t\t\t\t\t\t\ttask_runner,\n\t\t\t\t\t\t\ttask_epoch,\n\t\t\t\t\t\t\tlast_heartbeat,\n\t\t\t\t\t\t\tresult_code,\n\t\t\t\t\t\t\terror_msg,\n\t\t\t\t\t\t\tcreate_at,\n\t\t\t\t\t\t\tend_at \n\t\t\t\t\t\tfrom mo_task.sys_async_task where task_id>0 AND task_runner='00000000-0000-0000-0000-200000000000' order by task_id limit 1","session_info":"connectionId 1014"}
{"level":"ERROR","time":"2022/11/12 18:13:24.652815 +0000","name":"cn-service.task-runner","caller":"taskservice/task_runner.go:283","msg":"fetch task failed","uuid":"00000000-0000-0000-0000-200000000000","runner-id":"00000000-0000-0000-0000-200000000000","error":"context deadline exceeded"}
{"level":"ERROR","time":"2022/11/12 18:13:26.384749 +0000","caller":"frontend/util.go:505","msg":"operation error S3: GetObject, context canceled","session_info":"connectionId 1014","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logError\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:505\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3062\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3141\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:160"}
{"level":"ERROR","time":"2022/11/12 18:13:26.384866 +0000","caller":"frontend/util.go:505","msg":"query trace status","connection_id":1014,"statement":"select sys_async_task.task_id, sys_async_task.task_metadata_id, sys_async_task.task_metadata_executor, sys_async_task.task_metadata_context, sys_async_task.task_metadata_option, sys_async_task.task_parent_id, sys_async_task.task_status, sys_async_task.task_runner, sys_async_task.task_epoch, sys_async_task.last_heartbeat, sys_async_task.result_code, sys_async_task.error_msg, sys_async_task.create_at, sys_async_task.end_at from mo_task.sys_async_task where sys_async_task.task_id > 0 and sys_async_task.task_runner = 00000000-0000-0000-0000-200000000000 order by sys_async_task.task_id limit 1","status":"fail","error":"operation error S3: GetObject, context canceled","session_info":"connectionId 1014","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logError\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:505\ngithub.com/matrixorigin/matrixone/pkg/frontend.logStatementStringStatus\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:489\ngithub.com/matrixorigin/matrixone/pkg/frontend.logStatementStatus\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:481\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3070\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3141\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:160"}
{"level":"ERROR","time":"2022/11/12 18:13:26.384962 +0000","caller":"frontend/util.go:520","msg":"routine send response failed &{1 0 3 0xc00634e690 0 0 0}. error:illegal state  connectionId 1014","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logErrorf\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:520\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:166"}
{"level":"ERROR","time":"2022/11/12 18:13:26.384994 +0000","caller":"frontend/util.go:520","msg":"rollback the txn. connectionId 1014","stacktrace":"github.com/matrixorigin/matrixone/pkg/frontend.logErrorf\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/util.go:520\ngithub.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop\n\t/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:180"}
{"level":"INFO","time":"2022/11/12 18:13:34.655773 +0000","caller":"frontend/util.go:510","msg":"check password succeeded connectionId 1015"}
{"level":"INFO","time":"2022/11/12 18:13:34.655942 +0000","caller":"frontend/util.go:495","msg":"query trace","connection_id":1015,"query":"use mo_task","session_info":"connectionId 1015"}
{"level":"INFO","time":"2022/11/12 18:13:34.659188 +0000","caller":"frontend/util.go:510","msg":"User 36a76541-93c4-41a4-9b43-a0ac0c0ca4c4 change database from [] to [mo_task] connectionId 1015"}
```

### Steps to Reproduce

```Markdown
same with #6529
```

**Comment by daviszhen** [comment]
it is related to #6529

**Comment by yingfeng** [comment]
related to s3 bvt timeout which is tracked in #6209, and will keep observing

**Comment by daviszhen** [comment]
It is related to #6529 .

**Comment by aressu1985** [comment]
fixed: 0.6.0-1786399
